app/main.py

```diff
@@ -1,3 +1,84 @@
+from __future__ import annotations
```

Comment:
The import statement `from __future__ import annotations` should be placed at the top of the file without any trailing slashes or comments. Ensure it is correctly formatted as `from __future__ import annotations`.